lms/djangoapps/courseware/date_summary.py

```diff
@@ -240,7 +240,11 @@ class CourseStartDate(DateSummary):
 
     @property
     def date(self):
-        return self.course.start
+        enrollment = CourseEnrollment.get_enrollment(self.user, self.course_id)
+        if self.course.self_paced and enrollment and (enrollment.created > self.course.start):
```

Comment:
Might be worth early-exiting if `not self.course.self_paced` to avoid looking up something in the database that we don't need to. It might be clearer what you're doing here if you returned `max(created, start)` as well (instead of checking that as part of the conditional), but that's a matter of taste.

Reply:
+1